picard/script/functions.py

```diff
@@ -1370,7 +1372,7 @@ def func_unique(parser, multi, case_sensitive="", separator=MULTI_VALUED_JOINER)
 
 
 @script_function(documentation=N_(
-    """`$countryname(country_code, translate="")`
+    """`$countryname(country_code,translate="")`
```

Comment:
This change is unrelated to this PR, I committed it as https://github.com/metabrainz/picard/commit/40a03a14f5df143e2831d3032f94d46f93306b6c